src/pages/[domain]/dashboard.tsx

```diff
@@ -91,7 +150,31 @@ function Dashboard() {
           )}
           {!isBuilderPlaceOwner && (
             <>
-              <div className='mb-12'>
+              <EmailModal show={show} setShow={setShow} />
+              {!workerData?.emailVerified && (
```

Comment:
create new component VerifyEmailNotification that will have: 
- callback logic 
- render
- data needed (workerdata..)
- display condition

same ActiveDelegationNotification